pylint_odoo/checkers/modules_odoo.py

```diff
@@ -569,7 +569,8 @@ def _get_duplicate_xml_fields(self, fields):
         all_fields = {}
         for field in fields:
             field_xml = field.attrib.get('name')
-            if not field_xml:
+            if (not field_xml or (field.getparent() and
+                                  field.getparent().tag == 'search')):
```

Comment:
What about if is a inherited search view?

Shouldn't we skip for xpath node too?

Reply:
@moylop260 
You can show me the code to support this case to adding this code to the repository ?

Reply:
By example the following view is a inherited search view but don't have the tag `search` (Just use `xpath`):

https://github.com/odoo/odoo/blob/1f39b6021e08d471bc9f59162973e3297a808811/addons/pos_sale/views/sales_team_views.xml#L23

The main issue here is that if we skip `xpath` node too then we will have many false green results.

What about detect a duplicated field if there is same context and filter_domain?

| No | field name | context            | filter domain            |
|---|------------|--------------------|--------------------------|
| 1 | field1     | {}                 | []                       |
| 2 | field1     | {}                 | []                       |
| 3 | field1     | {'key1': 'value1'} | []                       |
| 4 | field1     | {'key1': 'value1'} | []                       |
| 5 | field1     | {}                 | [('field', '=' 'value')] |
| 6 | field1     | {}                 | [('field', '=' 'value')] |
| 7 | field1     | {'key1': 'value1'} | [('field', '=' 'value')] |

Duplicated:
 - 1 with 2
 - 3 with 4
 - 5 with 6

7 is not duplicated because have different context and domain

Reply:
@moylop260 
I thinks is more easy detect the name, context and the filter_doman 
I will made the changes necessary to detect this case